docs/samples/tensorrt/README.md

```diff
@@ -0,0 +1,65 @@
+
+# Predict on a KFService using TensorRT Inference Server
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/docs/serving/outbound-network-access/)
+
+
+> Knative is not able to resolve containers from the NVIDIA container registry.
+To work around this you need to skip resolution for nvcr.io by editing knative config-deployment.yaml. See details here: https://github.com/knative/serving/issues/4355
+## Create the KFService
+Apply the CRD
+```
+kubectl apply -f tensorrt.yaml 
+```
+
+Expected Output
+```
+kfservice.serving.kubeflow.org/simple-string created
+```
+
+## Run a prediction
+Uses the client at: https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-guide/docs/client.html#section-client-api
+
+
+1. get the cluster IP
+```
+CLUSTER_IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo $CLUSTER_IP
+```
+2. check server status
+```
+curl -H "Host: simple-string.default.example.com" http://${CLUSTER_IP}/api/status
+```
+3. edit /etc/hosts to map the CLUSTER IP to simple-string.default.example.com
```

Comment:
O.O!!

Reply:
Yeah - well the NVIDIA client is a C++ client that doesn't send host headers... :-( 
I can open up a separate PR/issue there but dont want to couple the two

Reply:
#163

Reply:
Opened an issue for header support for the NVIDIA clients in the NVIDIA tensorrt repo: https://github.com/NVIDIA/tensorrt-inference-server/issues/386